pkgs/by-name/gh/ghdl/test-simple.nix

```
@@ -1,19 +1,22 @@
-{ stdenv, ghdl-llvm, ghdl-mcode, backend }:
+{ stdenv, lib, ghdl-llvm, ghdl-mcode, ghdl-gcc, backend }:
 
 let
-  ghdl = if backend == "llvm" then ghdl-llvm else ghdl-mcode;
+  ghdl = if backend == "llvm" then ghdl-llvm else (if backend == "gcc" then ghdl-gcc else ghdl-mcode);
```

Nix' syntax is designed such that this will do the expected thing even without parentheses:
```suggestion
  ghdl = if backend == "llvm" then ghdl-llvm else if backend == "gcc" then ghdl-gcc else ghdl-mcode;
```